l10ninsp/slave.py

```diff
@@ -123,6 +132,19 @@ def finished(self, *args):
       rc = FAILURE
     self.sendStatus({'rc': rc})
 
+def compareDirs(path1, path2):
+    cljsPath = '/home/elmo/elmo/slave-ball/vendor-local/compare-locales.js'
```

Comment:
The path to compare-locales can't be hard-coded, can you use a relative path here?